src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -400,6 +400,11 @@ private void processNamedArgument(
                 argBindings.put("minRecValue", "NA");
                 argBindings.put("maxRecValue", "NA");
             }
+
+            // Add in the number of times you can specify it:
+            argBindings.put("minElements", argDef.field.getAnnotation(Argument.class).minElements());
```

Comment:
Can you open a different PR for this change? It seems independent from the bash-completion and it will be useful for the normal doclet and this PR will take longer...